Aztec/Classes/TextKit/TextViewPasteboardDelegate.swift

```diff
@@ -80,6 +80,21 @@ open class AztecTextViewPasteboardDelegate: TextViewPasteboardDelegate {
         return true
     }
 
+    private func fixColors(in string: NSAttributedString, using baseColor: UIColor?) -> NSAttributedString {
+        guard #available(iOS 13.0, *) else {
+            return string
+        }
+        let colorToUse = baseColor ?? UIColor.label
+
+        let newString = NSMutableAttributedString(attributedString: string)
+        newString.enumerateAttributes(in: newString.rangeOfEntireString, options: []) { (attributes, range, stop) in
+            if attributes[.foregroundColor] == nil {
+                newString.setAttributes([.foregroundColor: colorToUse], range: range)
```

Comment:
Out of interest, do we need to set color attributes, or can we just remove any that already exist there?

Reply:
From my investigation we need to set them explicitly or else we will get
the bug!